### PR TITLE
Allow for dual stack services to use when enabling experimental dual …

### PR DESCRIPTION
**Pull request by jacob-delgado**
…stack support

**Please provide a description of this PR:**

Part of the blog https://github.com/istio/istio.io/pull/12253 will reference these files to allow for user experimentation to provide feedback for developers doing dual stack work.

### CODE REVIEW

File: samples/tcp-echo/tcp-echo-dual-stack.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+metadata:
+  name: tcp-echo
+  labels:
+    app: tcp-echo
+    service: tcp-echo
+spec:
+  ipFamilyPolicy: RequireDualStack
+  ipFamilies:
+  - IPv6
+  - IPv4
```
Comment (jacob-delgado) [inline samples/tcp-echo/tcp-echo-dual-stack.yaml:29]: this is the relevant part

File: samples/tcp-echo/tcp-echo-ipv6.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+kind: Service
+metadata:
+  name: tcp-echo
+  labels:
+    app: tcp-echo
+    service: tcp-echo
+spec:
+  ipFamilyPolicy: SingleStack
+  ipFamilies:
+  - IPv6
```
Comment (jacob-delgado) [inline samples/tcp-echo/tcp-echo-ipv6.yaml:28]: this is only to be used for validation that single stack still works when using are doing their testing

**Review by zhlsunshine — APPROVED** [review]
LGTM

File: samples/tcp-echo/tcp-echo-dual-stack.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Istio Authors
```
Comment (kfaseela) [inline samples/tcp-echo/tcp-echo-dual-stack.yaml:1]:
```suggestion
# Copyright Istio Authors
```

File: samples/tcp-echo/tcp-echo-ipv4.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 Istio Authors
```
Comment (kfaseela) [inline samples/tcp-echo/tcp-echo-ipv4.yaml:1]:
```suggestion
# Copyright Istio Authors
```

File: samples/tcp-echo/tcp-echo-ipv6.yaml [hunk 5]
```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 Istio Authors
```
Comment (kfaseela) [inline samples/tcp-echo/tcp-echo-ipv6.yaml:1]:
```suggestion
# Copyright Istio Authors
```

**Review by kfaseela — APPROVED** [review]

**Pull request merged**